debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -96,4 +105,86 @@ public void enableTableCdc(String name) throws SQLException {
         String generateWrapperFunctionsStmts = CDC_WRAPPERS_DML.replaceAll(STATEMENTS_PLACEHOLDER, name);
         execute(enableCdcForTableStmt, generateWrapperFunctionsStmts);
     }
+
+    /**
+     * @return the current largest log sequence number
+     */
+    public Lsn getMaxLsn() throws SQLException {
+        final String LSN_COUNT_ERROR = "Maximum LSN query must return exactly one value";
+        return queryAndMap(GET_MAX_LSN, rs -> {
```

Comment:
Should we add something like `queryAndMapSingleResult()` which encapsulates the error handling in case there's more than one row?